src/MoBi.Presentation/Presenter/ParameterValuesPresenter.cs

```diff
@@ -80,5 +92,22 @@ public void UpdateDimension(ParameterValueDTO valueObject, IDimension newDimensi
 
          AddCommand(macroCommand);
       }
+
+      public void ShowContextMenu(IViewItem objectRequestingPopup, Point popupLocation) =>
+         _viewItemContextMenuFactory.CreateFor(objectRequestingPopup, this).Show(_view, popupLocation);
+
+      public void AddNewParameterValue()
+      {
+         _modalPresenter.Text = AppConstants.Captions.SelectLocalReferencePoint;
```

Comment:
I think this caption can change too. We are selecting a parameter, so it can probably be something simple like that.